zero_bin/rpc/src/native/txn.rs

```diff
@@ -163,6 +196,7 @@ async fn process_tx_traces(
 
         let balance = post_state.and_then(|x| x.balance.map(Compat::compat));
         let (storage_read, storage_written) = process_storage(
+            // Q: doesn't this remove the address from warm addresses?
```

Comment:
The prover extracts the access list (if any) from the txn RLP so it is harmless to remove these keys here

Reply:
Alright.